frontend/src/component/menu/Header/Header.tsx

```diff
@@ -171,12 +172,17 @@ const Header: VFC = () => {
     const routes = getRoutes();
     const adminRoutes = useAdminRoutes();
 
+    const excludeInsights = (r: INavigationMenuItem) =>
```

Comment:
I think we already have some kind of filtering in place, with `IRouteMenu` configuration